tests/multi_objective_tests/test_dominates.py

```diff
@@ -0,0 +1,99 @@
+from typing import List
```

Comment:
The `_dominates` function is defined on `optuna/study/_multi_objective.py`. Following the test file naming convention, how about writing this test code in `tests/study_tests/test_multi_objective.py`?